packages/rum/src/boot/recorderApi.spec.ts

```diff
@@ -37,11 +37,17 @@ describe('makeRecorderApi', () => {
 
     lifeCycle = new LifeCycle()
     stopRecordingSpy = jasmine.createSpy('stopRecording')
-    startRecordingSpy = jasmine.createSpy('startRecording').and.callFake(() => ({
-      stop: stopRecordingSpy,
-    }))
+    startRecordingSpy = jasmine.createSpy('startRecording')
+
+    // Workaround because using resolveTo(startRecordingSpy) is not was not working
```

Comment:
```suggestion
    // Workaround because using resolveTo(startRecordingSpy) was not working
```

Reply:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ You resolved this but I don't understand what `is not was not` means if it's not a typo.

Reply:
My bad, sry